src/components/ReportViewer.js

```diff
@@ -47,7 +47,7 @@ class ReportViewer extends Component {
     return (
       <div className="reportViewer">
         <Toolbar className="reportViewerToolbar">
-          <Button className="backToExplore" onClick={() => history.push('/')}> <KeyboardArrowLeft/>Explore</Button>
+          <Button className="backToExplore" onClick={() => history.goBack()}> <KeyboardArrowLeft/>Explore</Button>
```

Comment:
Yeah I agree that since this is going back we should call it "Back" instead of "Explore"